services/frontend/src/scenes/Profile/ProfileLayout.js

```diff
@@ -86,32 +88,58 @@ const renderSectionComponent = (sectionName, sectionProps) => {
     }
 };
 
-const PersonalDetails = ({profile}) => (
-    <Paper className="profile-card profile-card-personal-details">
-        <AvatarIcon name={profile.name} personsRole="" className="profile-avatar-icon" />
-        <div className="profile-card-content">
-            <h2 className="profile-card-title">
-                {profile.name}
-            </h2>
-
-            <h3 className="profile-card-subtitle">
-                {profile.primaryRole}
-            </h3>
-
-            <div className="profile-card-contact">
-                <p className="profile-card-text">
-                    <Email />
-                    {profile.contactEmail}
-                </p>
-
-                <p className="profile-card-text">
-                    <LocalPhone />
-                    {profile.phone}
-                </p>
-            </div>
-        </div>
-    </Paper>
-);
+const PersonalDetails = ({profile}) => {
+    const [personalDetailsDialogOpen, setPersonalDetailsDialogOpen] = React.useState(false);
+
+    function openDialog() {
```

Comment:
Are you purposely using regular functions just to check if I'm actually reviewing your code or something?...

Reply:
Totally,

Swapped to arrow functions.